heron/scheduler-core/tests/java/com/twitter/heron/scheduler/dryrun/Constants.java

```diff
@@ -0,0 +1,25 @@
+// Copyright 2016 Twitter. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.twitter.heron.scheduler.dryrun;
+
+public final class Constants {
+
+  public static final String TEST_DATA_PATH =
+      "/__main__/heron/scheduler-core/tests/java/com/twitter/heron/scheduler/dryrun/testdata";
```

Comment:
The data files should be listed as bazel test resources, which will put them in the classpath of the unit test, so in each test you can just reference them relative to the tests/resources directory.